### PR TITLE
[ios][contacts] Migrate to Expo Modules

### PR DESCRIPTION
**Pull request by alanjhughes**
# Why
Closes ENG-10691
One of the last modules to be migrate

# How
Usual migration steps.

# Test Plan
bare-expo and NCL

### CODE REVIEW

**Comment by fobos531** [comment]
Very excited for this one! It will definitely help building this: https://github.com/expo/expo/issues/2645 🔥

**Comment by alanjhughes** [comment]
Hey @fobos531, no, it just needs to be reviewed.

**Comment by fobos531** [comment]
> Hey @fobos531, no, it just needs to be reviewed.

Okay, that is good to hear. If I want to contribute something to expo-contacts (both for iOS and Android) at this point, should I fork from your branch (@alanhughes/ios/contacts) or wait until this gets merged?

**Review by lukmccall — APPROVED** [review]

**Review by tsapeta — APPROVED** [review]

File: packages/expo-contacts/src/Contacts.ts
Comment (lukmccall) [reply packages/expo-contacts/src/Contacts.ts:650]: It may not work on Android 😅

**Pull request merged**
